src/main/resources/application.properties

```diff
@@ -47,7 +47,7 @@ export.client.registration.retry.attempts=100
 #logging levels (used to control log4j entries)
 logging.level.org.springframework=ERROR
 logging.level.org.apache=ERROR
-logging.level.org.edgexfoundry=INFO
+logging.level.org.edgexfoundry=DEBUG
```

Comment:
LogLevel default is usually INFO. Is this change intended?

Reply:
Nope -  another JW oops